src/styles/page.css

```
@@ -0,0 +1,3 @@
+.page {
```

```suggestion
.html {
```

rename on 'normalize'